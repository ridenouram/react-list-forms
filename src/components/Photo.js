import React from 'react';
import PropTypes from 'prop-types';
import styles from './style.css';

export default function Photo({ photo }) {
  return (
    <img className={styles.Photo} src={photo} />
  );
}

Photo.propTypes = {
  photo: PropTypes.string.isRequired
};
