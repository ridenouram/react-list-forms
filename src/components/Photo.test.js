import React from 'react';
import { shallow } from 'enzyme';
import Photo from './Photo';

describe('photo component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Photo photo='' />);
    expect(wrapper).toMatchSnapshot();
  });
});
