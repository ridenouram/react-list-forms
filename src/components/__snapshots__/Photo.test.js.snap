// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`photo component matches snapshot 1`] = `
<img
  className="Photo"
  src=""
/>
`;
